Flaky Redis lock test

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

Flaky Redis lock test

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

1. Run CI
2. Flip a coin

<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!-- Please paste terminal output, upload logs (as .txt) or upload screenshots. -->

```
--- FAIL: TestMutexLock (1.05s)
##[error]    redis_test.go:831: LockMutex returned before previous caller unlocked: %!s(<nil>)
FAIL
FAIL	go.thethings.network/lorawan-stack/v3/pkg/redis	1.194s
```

#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

No test failure

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

GitHub Actions

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

Not sure

#### How do you propose to test this?
<!-- Please think about how this is verified as fixed. -->

Not sure

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

No

--- a/pkg/redis/redis_test.go
+++ b/pkg/redis/redis_test.go
@@ -781,19 +781,19 @@ func TestProtoDeduplicator(t *testing.T) {
 	a.So(listTTL, should.BeLessThanOrEqualTo, ttl)
 }
 
-func TestMutexLock(t *testing.T) {
+func TestMutex(t *testing.T) {
 	a := assertions.New(t)
 
 	ctx := test.ContextWithT(test.Context(), t)
 	ctx = log.NewContext(ctx, test.GetLogger(t))
-	ctx, cancel := context.WithTimeout(ctx, 2*time.Second+(1<<6)*test.Delay)
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Second+(1<<8)*test.Delay)
 	defer cancel()
 
 	cl, flush := test.NewRedis(t, "redis_test")
 	defer flush()
 	defer cl.Close()
 
-	ttl := (1 << 6) * test.Delay
+	ttl := (1 << 8) * test.Delay
 	key := cl.Key("test1")
 
 	err := LockMutex(ctx, cl, key, "test-id-1", ttl)

